lib/core/widgets/app_text_form_field.dart

```diff
@@ -0,0 +1,189 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+import '../theming/colors.dart';
+
+// ignore: must_be_immutable
```

Comment:
don't make this : // ignore: must_be_immutable

fix the issue, don't ignore it.

and I feel this widget is a little bit larger than normal ? consider not using all these attributes.